### DISCUSSION TITLE
When injected it keeps crashing rdr2

### DISCUSSION THREAD

**Discussion by TexasToast03** [YimMenu/HorseMenu#186] [Q&A] [answered]
any way to fix this?

**Comment by tuyilmaz** [comment]
Submit your crash log, so we can see what crashed your game. By the way discussion is not the right place for issues. Please open an issue.

**Reply by TexasToast03** [reply, marked as answer]
I run windows 10 and I had to turn off my notifications (bottom right) and it worked.

**Reply by TexasToast03** [reply]
> Submit your crash log, so we can see what crashed your game. By the way discussion is not the right place for issues. Please open an issue.

Where do I find it at
